modules/copy-template/build/copy-template/Dockerfile

```diff
@@ -1,14 +1,10 @@
 FROM registry.access.redhat.com/ubi9/ubi-minimal AS builder
-RUN microdnf install -y tar gzip && microdnf clean all
+RUN microdnf install -y tar gzip golang && microdnf clean all
```

Comment:
we download golang from official site, because many times the newest release is not available in ubi image.

Reply:
then maybe better to use a golang builder image like "registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13 AS builder "?  (just an example, could use newer )

Reply:
this should save the the special code for arch aware downloads. 
Those tasks are fairly small and short lived commands and they should be okay not to use the latest golang release. Unless you are waiting for a new feature?

Reply:
also noticed I could remove the installation of tar and gzip

Reply:
I like the approach with registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13 base image.

Reply:
registry.ci seems authenticated. There is an open, unauthenticated and maintained go builder based on ubi already
which is multi-arch and its a great drop-in replacement - registry.access.redhat.com/ubi9/go-toolset - I'll use that

Reply:
sounds good